### PR TITLE
Completed Task opt

### PR DESCRIPTION
**Pull request by benaadams**

### CODE REVIEW

**Comment by davidfowl** [comment]
:shipit:

File: src/Microsoft.AspNet.Server.Kestrel/Infrastructure/TaskUtilities.cs [hunk 1]
```diff
@@ -7,13 +7,10 @@ namespace Microsoft.AspNet.Server.Kestrel.Infrastructure
...
     {
-        public static Task CompletedTask = NewCompletedTask();
-
-        private static Task NewCompletedTask()
-        {
-            var tcs = new TaskCompletionSource<int>();
-            tcs.SetResult(0);
-            return tcs.Task;
-        }
+#if DNXCORE50
```
Comment (davidfowl) [inline src/Microsoft.AspNet.Server.Kestrel/Infrastructure/TaskUtilities.cs]: Needs DOTNET5_4

Comment (benaadams) [inline src/Microsoft.AspNet.Server.Kestrel/Infrastructure/TaskUtilities.cs]: Changed, but failed on Travis post tests time out; will keep spinning the wheel till it passes.

**Comment by benaadams** [comment]
Just recording CI submission results:

1. Travis Pass
-- Update define const
2. Travis Timeout
3. Travis SIGSEGV uv_async_send
4. Travis Timeout 
5. Pass

**Pull request merged**